pom.xml

```diff
@@ -145,5 +169,61 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>release-sign-artifacts</id>
```

Comment:
@btyoshikawaa 
I renamed them. 8ef9712ea8c998da2ceb2ea2cc04217db9605e20.
and sorted of profiles. e7631b74bb5c8ecccd574cfa27df51f93be0f09d.